components/world-4/breeding/pets.tsx

```diff
@@ -0,0 +1,522 @@
+
+                        return (
+                            petInFenceyard > 0 ?
+                                <ShadowBox background="dark-1" border={{ size: '2px', style: 'solid', color: 'green' }} key={pIndex} direction="row" gap="medium" margin={{ bottom: 'medium', right: 'small' }} align="center" pad="small" style={{ opacity: pet.shinyLevel > 0 ? 1 : .5 }}>
+                                    <TipDisplay
+                                        size='medium'
+                                        maxWidth='medium'
+                                        heading={enemy?.data.details.Name ?? "Unknown"}
+                                        body={
+                                            <Box>
+                                                <Text size="medium">Currently in fenceyard : {petInFenceyard}</Text>
+                                                <Text size="medium">Total contribution per day : {nFormatter(petInFenceyard * breeding.getShinySpeed(starSignEquipped, silkRodeChip))}</Text>
+                                                {timeToNextLevel > 0 && <Text>Time to next level: {toTime(timeToNextLevel)}</Text>}
+                                            </Box>
+                                        }
+                                        direction={TipDirection.Down} 
+                                    >
+                                        <IconImage data={{ location: enemy?.id.toLowerCase() ?? "Unknown", width: 67, height: 67 }} style={{ paddingBottom: '15px' }} />
+                                    </TipDisplay>
+                                    <Grid columns={["50%", "50%"]} fill align="center">
+                                        <Box>
+                                            <Text size="16px">Lvl: {pet.shinyLevel}</Text>
+                                            <Text size="16px">{Math.floor(pet.shinyProgress)}/{pet.getNextShinyGoal()} days</Text>
+                                        </Box>
+                                        <Text size="16px">{pet.getShinyText()}</Text>
+                                    </Grid>
+                                </ShadowBox>
+                                :
+                                <ShadowBox background="dark-1" key={pIndex} direction="row" gap="medium" margin={{ bottom: 'medium', right: 'small' }} align="center" pad="small" style={{ opacity: pet.shinyLevel > 0 ? 1 : .5 }}>
+                                    <IconImage data={{ location: enemy?.id.toLowerCase() ?? "Unknown", width: 67, height: 67 }} style={{ paddingBottom: '15px' }} />
+                                    <Grid columns={["50%", "50%"]} fill align="center">
+                                        <Box>
+                                            <Text size="16px">Lvl: {pet.shinyLevel}</Text>
+                                            <Text size="16px">{Math.floor(pet.shinyProgress)}/{pet.getNextShinyGoal()} days</Text>
```

Comment:
I didn't think about this before but are these numbers actually useful? like with the speed multipliers it's not really 700 or whatever days to get a level right?

Also if you have multiples of the same pet in the fenceyard that further cuts it down right?

I don't see how showing these numbers gives people a good indication of time. What do you think?

Reply:
Yeah right now it's mostly just displaying the game data and not calculating how many real days it'd take

I guess I could improve it for both Shiny and Breedability by taking it into account

Reply:
I guess I already was taking it into account, for pets that are in the fence yard (those having a green border) you can hover the pet icon and it'll tell you more details, like how many time exactly you need to wait for next level given how many of this pet are in the fence yard and your multiplier

I guess I could make it more visible, like adding an "information icon" instead of letting user guess they have to hover

Reply:
I made some modifications to the UI, should be better this way